src/server/start-server.ts

```diff
@@ -1,20 +1,51 @@
+import { AxiosResponse } from "axios";
 import config from "config";
+import RpcMethod from "iotex-antenna/lib/rpc-method/node-rpc-method";
 // @ts-ignore
-import { Server } from "onefx/lib/server";
+import { Config, Server } from "onefx/lib/server";
+import { AntennaResolver } from "../api-gateway/resolvers/antenna";
+import { MetaResolver } from "../api-gateway/resolvers/meta";
 import { setModel } from "../model";
 import "../shared/common/setup-big-number";
 import { setGateways } from "./gateways/gateways";
 import { setMiddleware } from "./middleware";
 import { setServerRoutes } from "./server-routes";
 
-export async function startServer(): Promise<void> {
+export type MyServer = Server & {
```

Comment:
It's unnecessary to define `MyServer` type and eventually end up with `ts-ignore` like this. Instead, you can just adapt the `config` to `Config` type by spreading default value like below:

```suggestion
const defaultConfig: Config = {
  project: "",
  server: {
    host: "",
    port: "",
    staticDir: "",
    delayInitMiddleware: false,
    cookie: {
      secrets: []
    },
    noSecurityHeadersRoutes: {},
    noCsrfRoutes: {}
  },
  gateways: {
    logger: {
      enabled: false,
      baseDir: "",
      topicName: "",
      level: "debug"
    }
  },
  csp: {},
  analytics: {},
  session: {}
};

const serverConfig: Config = {
  ...defaultConfig,
  ...config
};

export async function startServer(): Promise<Server> {
  const server = new Server(serverConfig);
  setGateways(server);
  setMiddleware(server);
  setModel(server);
  setServerRoutes(server);

  const port = process.env.PORT || config.get("server.port");

  server.listen(parseInt(`${port}`, 10));
  return server;
}
```